### PR TITLE
Yolov7Face add predict's python Interface

### PR DESCRIPTION
**Pull request by CoolKbh**
### Issue
resolve https://github.com/PaddlePaddle/FastDeploy/issues/1030 

### Description
在/fast deploy/vision.h中class YOLOv7Face(FastDeployModel):下增加predict接口

### CODE REVIEW

File: python/fastdeploy/vision/facedet/contrib/yolov7face.py [hunk 1]
```diff
@@ -140,6 +140,17 @@ def __init__(self,
 
         assert self.initialized, "YOLOv7Face initialize failed."
 
+    def predict(self, input_image, conf_threshold=0.5, nms_iou_threshold=0.45):
```
Comment (jiangjiajun) [inline python/fastdeploy/vision/facedet/contrib/yolov7face.py]: 先前已经确认后面predict只接收image，不再附加其它参数。 conf_threshold, nms_iou_threshold都改为调用preprocessor和postprocessor来修改

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: python/fastdeploy/vision/facedet/contrib/yolov7face.py (hunk 1 above)
Comment (CoolKbh) [reply python/fastdeploy/vision/facedet/contrib/yolov7face.py]: 好的

File: python/fastdeploy/vision/facedet/contrib/yolov7face.py [hunk 2]
```diff
@@ -140,6 +140,13 @@ def __init__(self,
 
         assert self.initialized, "YOLOv7Face initialize failed."
 
+    def predict(self, input_image,):
```
Comment (wjj19950828) [inline python/fastdeploy/vision/facedet/contrib/yolov7face.py]: 逗号去掉

**Review by wjj19950828 — CHANGES_REQUESTED** [review]

File: python/fastdeploy/vision/facedet/contrib/yolov7face.py (hunk 2 above)
Comment (CoolKbh) [reply python/fastdeploy/vision/facedet/contrib/yolov7face.py]: done

**Review by wjj19950828 — APPROVED** [review]

**Pull request merged**